src/System.Diagnostics.Debug/tests/System.Diagnostics.Debug.Tests.csproj

```diff
@@ -30,6 +30,7 @@
     </ProjectReference>
     <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj" />
     <ProjectReference Include="..\..\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj" />
+    <TargetingPackReference Include="System.Private.CoreLib" Condition="'$(TestTFM)' != 'net46'" />
```

Comment:
We shouldn't condition things like this in the test project on TestTFM. If we need to make this test project have a TargetGroup net46 configuration. It is just pure accident that these ever worked when running on net46 (assuming they actually do work there).